### ISSUE TITLE
missing concept ids for ACT

### ISSUE THREAD

**Issue by stephanieshong** [National-COVID-Cohort-Collaborative/Data-Ingestion-and-Harmonization#69]
The missing qualifiers will have 0 concepts ids until one is assigned 
   Pink
   KNEE
   BLOODY
   White
   P21950
   BLOOD
   Marked
   NASAL
   Tube 2
   CLOUDY
   RED
   Rouleaux
   FLUID
   HEART
   Du
   Serum
   P21967
   CLEAR
   GREEN
   ABSCESS
   RENAL

**Comment by mim18** [comment]
These will be difficult to determine without knowing the concept_cd. colors can be for sputum, urine, feces and I believe each has its own omop concept_id. We will need to create some translation code that will use concept_cd to translate

**Comment by stephanieshong** [comment]
Agree. We will evaluate this over time and apply the translation code case by case. Thank you.
